.github/workflows/ci.yml

```
@@ -8,27 +8,27 @@ on:
       - '*'
 jobs:
   mix_format:
-    name: mix format (Elixir 1.10.2 OTP 22.3)
+    name: mix format (Elixir 1.12.0 OTP 24.0.1)
```

```suggestion
    name: mix format (Elixir 1.12.1 OTP 24.0.1)
```

Acho que já rola irmos para 12.1. Oq acha?

---

Done 👍🏽